tests/Microsoft.Identity.Test.Unit/TestBase.cs

```diff
@@ -48,6 +60,15 @@ public virtual void TestInitialize()
         [TestCleanup]
         public virtual void TestCleanup()
         {
+            if (ManagedIdentityEnabled)
```

Comment:
not needed, this should be part of a "ResetForTest" API. @gladjohn - who is writing the ResetForTest API?